modules/dreamview/backend/handlers/image_handler.cc

```diff
@@ -78,11 +78,8 @@ void ImageHandler::OnImage(const sensor_msgs::CompressedImage &image) {
 }
 
 ImageHandler::ImageHandler() : requests_(0) {
-  if (FLAGS_use_navigation_mode) {
```

Comment:
We don't want to have both subscribers on as there are both the two cameras on the car and only one of them should be displayed depending on mode.

Reply:
@mickeyouyou This has been fixed by another PR, please resolve the conflict and squash all the commits into one, and then this PR is ready to merge.

Reply:
OK